options.lua

```diff
@@ -0,0 +1,69 @@
+local profile = Game.GetProfile()
+
+local function split_string(inputstr, sep)
```

Comment:
If `on_input_changed` is unused, then this looks like it becomes unused as well.